src/segment-loader.js

```diff
@@ -347,6 +347,8 @@ export default class SegmentLoader extends videojs.EventTarget {
     const id = initSegmentId(map);
     let storedMap = this.initSegments_[id];
 
+    // TODO: We should use the HTTP Expires header to invalidate our cache per
+    // https://tools.ietf.org/html/draft-pantos-http-live-streaming-23#section-6.2.3
```

Comment:
Whoops, this accidentally ended up in the `initSegment` method instead of `segmentKey`